src/components/common/ListRow/index.js

```diff
@@ -13,6 +13,7 @@ const ListRow = ({
   rightText,
   icon,
   showIcon,
+  suffix,
   children,
 }) => (
   <Fragment>
```

Comment:
If there's no need to pass a key to a Fragment, you can use the short sintax `<>....</>`